avocado/utils/software_manager.py

```diff
@@ -495,6 +499,39 @@ def provides(self, name):
             return None
 
 
+class DnfBackend(YumBackend):
+
+    """
+    Implements the dnf backend for software manager.
+
+    DNF is the successor to yum in recent Fedora.
+    """
+    def __init__(self):
+        """
+        Initializes the base command and the yum package repository.
+        """
+        super(DnfBackend, self).__init__()
+        executable = utils_path.find_command('dnf')
+        base_arguments = '-y'
+        self.base_command = executable + ' ' + base_arguments
+
+    def install(self, name):
+        """
+        Installs package [name]. Handles local installs.
+        """
+        command = 'install'
+        if os.path.isfile(name):
+            name = os.path.abspath(name)
```

Comment:
Yes, it does make sense. I can do that. I'll send an updated version tomorrow, thanks!